microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -469,9 +469,14 @@ def _install(self) -> bool:
         self.dpdk_path = self.node.working_path.joinpath(self._dpdk_repo_path_name)
 
         if self.find_testpmd_binary(
-            assert_on_fail=False
+            assert_on_fail=False, check_path="/usr/local/bin"
         ):  # tools are already installed
             return True
+        else:
+            # we are building dpdk on a node where it was installed
+            # as a package, this will make it unclear which version to use
+            # for the next test, so mark the environment dirty.
```

Comment:
It should be fine, I'll remove. Trying to avoid weird failures because of bad state.

Reply:
We should understand the bad state, and avoid overusing the "dirty" feature. The dirty feature will slow down the test runs, because it needs to deploy more environments.